app/Http/Controllers/Api/UnsubscribeController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Services\Manager;
+use Gladiator\Models\Competition;
+use Gladiator\Http\Requests\UnsubscribeRequest;
+
+class UnsubscribeController extends ApiController
+{
+    /**
+     * @var Gladiator\Services\Manager;
+     */
+    protected $manager;
+
+    public function unsubscribe(UnsubscribeRequest $request)
+    {
+        $credentials = $request->all();
+
+        $competition = Competition::find($credentials['competition_id']);
```

Comment:
Scratch that, you will need the model instance to call the `unsubscribe` method on it, but you don't need the activity to be attached to it. 

for example: 

```
$competition = Competition::find($data['competition_id']);
$competition->unsubscribe($data['northstar_id']);
```